spikingjelly/datasets/base.py

```
@@ -350,7 +373,8 @@ def __init__(self, cfg: NeuromorphicDatasetConfig, raw_root: Path):
 
         原始事件数据的数据集构建器。
 
-        此构建器不执行任何预处理，直接使用原始数据集作为处理后的数据集。每个样本通过 ``np.load`` 直接加载为原始事件文件（例如 ``.npz``），无需帧积分。
+        此构建器不执行任何预处理，直接使用原始数据集作为处理后的数据集。``.npy``
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)